en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/security/importing-certificates-to-the-choreo-connect-truststore.md

```diff
@@ -1,55 +0,0 @@
-# Importing Certificates to the API Choreo Connect Truststore
```

Comment:
@suksw , has this content been moved to another page? Trying to see if we need to add a redirection for this.

Reply:
In fact, this is the file I renamed as "Component certificates" and added more content. Therefore the content exists in  "Component certificates".

Reply:
starting from https://github.com/wso2/docs-apim/pull/5178/commits/84f203f3c5b883d926a83fc8a889761214a37744#diff-10df25db36bd90e540d4b02d0fc4746e69d2add665d8f7b4bf8ae5227e4928f3R31

Reply:
Ok so then we need to add a redirection for it so that people who have the old location can go to the new location

Reply:
actually, there's this too `...onnect/security/tls/transport-security.md → ...nect/security/tls/backend-certificates.md`. How do i add a redirection?